fluvii/kafka_tools/fluvii_toolbox.py

```diff
@@ -92,51 +93,107 @@ def create_topics(self, topic_config_dict, ignore_existing_topics=True):
                 futures[topic].result()
         LOGGER.info(f'Created topics: {list(topic_config_dict.keys())}')
 
-    def alter_topics(self, topic_config_dict, retain_configs=True, ignore_missing_topics=True, protected_configs=None):
+    def alter_topics(self, topic_config_dict, retain_configs=True, ignore_missing_topics=True, ignore_field_list=None):
```

Comment:
It would be more appropriate to rename this parameter to 'fields_to_ignore' to make its purpose clearer.

Reply:
@kevinkjt2000 thoughts on the name? I'm fine with it as-is

Reply:
I like having plural nouns for list variables and I find it cleaner to put adjectives in front. Types inside of a variable name are something that is avoidable. Consider `ignorable_fields` or `ignored_fields`, possibly with a type hint of `: Optional[List[str]]`

Reply:
I do like the typing, though I feel like that should maybe be a separate initiative in the library in general. I'll rename it to `ignorable_fields` for now.

Reply:
Renamed :+1: